docs/configuration/project-references.md

```diff
@@ -0,0 +1,147 @@
+Glint works out of the box with TypeScript’s [Project References][pr] feature:
+
+- Instead of running just `glint`, you run `glint --build`, and Glint will correctly do incremental builds for composite projects.
+- In the future, the Glint Language Server also takes advantage of composite projects to improve performance when working in just one part of a large codebase.
```

Comment:
I would guess this already works as expected? To my knowledge, vanilla `tsserver` never triggers a build in a composite project itself, it just implicitly takes advantage of `.d.ts` files that have been generated by `--build` to avoid having to typecheck the other projects in the composite graph.

Reply:
Ah, sweet. Noted. I'll update accordingly!

Reply:
28c7e8b, also added a note about WIP on the other flags.